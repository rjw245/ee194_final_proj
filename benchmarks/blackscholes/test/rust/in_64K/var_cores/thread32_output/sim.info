{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread32_output',
             '-n',
             '32',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../rust/bs',
             '32',
             '../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (12855.850613,
            28.80662,
            334028,
            0,
            0,
            0,
            517796,
            0,
            0,
            0,
            16760,
            0,
            0,
            0,
            510133,
            10391422),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=32 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/thread32_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../rust/bs 32 ../../../inputs/in_64K.txt'],
 't_elapsed': 17340.587493896484,
 't_start': 1460954529.489094,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 6970961920L}
