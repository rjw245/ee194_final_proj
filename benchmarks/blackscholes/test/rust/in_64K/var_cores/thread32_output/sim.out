                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7      | Core 8      | Core 9      | Core 10     | Core 11     | Core 12     | Core 13     | Core 14     | Core 15     | Core 16     | Core 17     | Core 18     | Core 19     | Core 20     | Core 21     | Core 22     | Core 23     | Core 24     | Core 25     | Core 26     | Core 27     | Core 28     | Core 29     | Core 30     | Core 31    
  Instructions                       |  4219544811 |      422084 |      191382 |      185913 |      189127 |      187274 |      189243 |      186520 |      188056 |      188566 |      187513 |      193048 |      187771 |      186988 |      188977 |      191506 |      188076 |      189959 |      190516 |      192116 |      188807 |      190314 |      189867 |      189214 |      189158 |      193195 |      188851 |      193074 |      187462 |      190339 |      190976 |      191809
  Cycles                             |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598 |  2471486598
  IPC                                |        1.71 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  Time (ns)                          |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300 |   929130300
  Idle time (ns)                     |     1310900 |   928987078 |   929067415 |   929071694 |   929062466 |   929064170 |   929066938 |   929067415 |   929051310 |   929056599 |   929057954 |   929062604 |   929058420 |   929042163 |   929067904 |   929068437 |   929061197 |   929037882 |   929065132 |   929066831 |   929056448 |   929068932 |   929051651 |   929052695 |   929054379 |   929049418 |   929067479 |   929062007 |   929056178 |   929051186 |   929065370 |   929057155
  Idle time (%)                      |        0.1% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0%
Branch predictor stats               |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
  num correct                        |   147474265 |       44723 |       20103 |       19550 |       19916 |       19723 |       19938 |       19652 |       19771 |       19891 |       19842 |       20582 |       19845 |       19724 |       20058 |       20418 |       19906 |       20130 |       20244 |       20495 |       20091 |       20273 |       20169 |       20083 |       20138 |       20690 |       20122 |       20646 |       19825 |       20068 |       20266 |       20253
  num incorrect                      |     6695545 |        4098 |        2069 |        1971 |        2007 |        1971 |        2005 |        1966 |        2011 |        2013 |        1971 |        2058 |        1973 |        1986 |        1997 |        2031 |        2004 |        2013 |        2031 |        2040 |        1972 |        1987 |        2011 |        1982 |        1989 |        2038 |        1959 |        2027 |        1998 |        1974 |        2031 |        2011
  misprediction rate                 |       4.34% |       8.39% |       9.33% |       9.16% |       9.15% |       9.09% |       9.14% |       9.09% |       9.23% |       9.19% |       9.04% |       9.09% |       9.04% |       9.15% |       9.05% |       9.05% |       9.15% |       9.09% |       9.12% |       9.05% |       8.94% |       8.93% |       9.07% |       8.98% |       8.99% |       8.97% |       8.87% |       8.94% |       9.16% |       8.96% |       9.11% |       9.03%
  mpki                               |        1.59 |        9.71 |       10.81 |       10.60 |       10.61 |       10.52 |       10.59 |       10.54 |       10.69 |       10.68 |       10.51 |       10.66 |       10.51 |       10.62 |       10.57 |       10.61 |       10.66 |       10.60 |       10.66 |       10.62 |       10.44 |       10.44 |       10.59 |       10.47 |       10.52 |       10.55 |       10.37 |       10.50 |       10.66 |       10.37 |       10.63 |       10.48
TLB Summary                          |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
    num accesses                     |   858652591 |       53646 |       25523 |       24720 |       25221 |       24974 |       25306 |       24832 |       25090 |       25196 |       25014 |       26004 |       25089 |       24921 |       25270 |       25728 |       25108 |       25412 |       25515 |       25836 |       25291 |       25520 |       25452 |       25285 |       25347 |       26020 |       25300 |       25955 |       24981 |       25494 |       25606 |       25727
    num misses                       |         209 |         114 |         105 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          99 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98 |          98
    miss rate                        |       0.00% |       0.21% |       0.41% |       0.40% |       0.39% |       0.39% |       0.39% |       0.39% |       0.39% |       0.39% |       0.39% |       0.38% |       0.39% |       0.39% |       0.39% |       0.38% |       0.39% |       0.39% |       0.38% |       0.38% |       0.39% |       0.38% |       0.39% |       0.39% |       0.39% |       0.38% |       0.39% |       0.38% |       0.39% |       0.38% |       0.38% |       0.38%
    mpki                             |        0.00 |        0.27 |        0.55 |        0.53 |        0.52 |        0.52 |        0.52 |        0.53 |        0.52 |        0.52 |        0.52 |        0.51 |        0.52 |        0.52 |        0.52 |        0.51 |        0.52 |        0.52 |        0.51 |        0.51 |        0.52 |        0.51 |        0.52 |        0.52 |        0.52 |        0.51 |        0.52 |        0.51 |        0.52 |        0.51 |        0.51 |        0.51
  D-TLB                              |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
    num accesses                     |  2814314482 |      144353 |       65698 |       63045 |       63875 |       63378 |       63944 |       63141 |       63589 |       63691 |       63293 |       64460 |       63329 |       63283 |       63544 |       64057 |       63406 |       63864 |       63947 |       64194 |       63391 |       63736 |       63763 |       63598 |       63508 |       64352 |       63380 |       64468 |       63279 |       65266 |       64053 |       65582
    num misses                       |      184181 |         409 |         159 |         111 |         106 |         125 |         110 |         116 |         110 |         156 |         125 |         125 |         129 |         132 |         130 |         123 |         173 |         114 |         120 |         173 |         125 |         143 |         127 |         143 |         147 |         151 |         140 |         145 |         154 |         154 |         154 |         151
    miss rate                        |       0.01% |       0.28% |       0.24% |       0.18% |       0.17% |       0.20% |       0.17% |       0.18% |       0.17% |       0.24% |       0.20% |       0.19% |       0.20% |       0.21% |       0.20% |       0.19% |       0.27% |       0.18% |       0.19% |       0.27% |       0.20% |       0.22% |       0.20% |       0.22% |       0.23% |       0.23% |       0.22% |       0.22% |       0.24% |       0.24% |       0.24% |       0.23%
    mpki                             |        0.04 |        0.97 |        0.83 |        0.60 |        0.56 |        0.67 |        0.58 |        0.62 |        0.58 |        0.83 |        0.67 |        0.65 |        0.69 |        0.71 |        0.69 |        0.64 |        0.92 |        0.60 |        0.63 |        0.90 |        0.66 |        0.75 |        0.67 |        0.76 |        0.78 |        0.78 |        0.74 |        0.75 |        0.82 |        0.81 |        0.81 |        0.79
  L2 TLB                             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
    num accesses                     |      184390 |         523 |         264 |         209 |         204 |         223 |         208 |         214 |         208 |         254 |         223 |         223 |         227 |         230 |         228 |         221 |         271 |         212 |         218 |         271 |         223 |         241 |         226 |         241 |         245 |         249 |         238 |         243 |         252 |         252 |         252 |         249
    num misses                       |         970 |         271 |         199 |         176 |         180 |         180 |         182 |         181 |         185 |         188 |         187 |         190 |         190 |         192 |         193 |         194 |         197 |         187 |         195 |         199 |         196 |         200 |         199 |         202 |         201 |         205 |         203 |         203 |         208 |         208 |         209 |         211
    miss rate                        |       0.53% |      51.82% |      75.38% |      84.21% |      88.24% |      80.72% |      87.50% |      84.58% |      88.94% |      74.02% |      83.86% |      85.20% |      83.70% |      83.48% |      84.65% |      87.78% |      72.69% |      88.21% |      89.45% |      73.43% |      87.89% |      82.99% |      88.05% |      83.82% |      82.04% |      82.33% |      85.29% |      83.54% |      82.54% |      82.54% |      82.94% |      84.74%
    mpki                             |        0.00 |        0.64 |        1.04 |        0.95 |        0.95 |        0.96 |        0.96 |        0.97 |        0.98 |        1.00 |        1.00 |        0.98 |        1.01 |        1.03 |        1.02 |        1.01 |        1.05 |        0.98 |        1.02 |        1.04 |        1.04 |        1.05 |        1.05 |        1.07 |        1.06 |        1.06 |        1.07 |        1.05 |        1.11 |        1.09 |        1.09 |        1.10
Cache Summary                        |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
    num cache accesses               |   858652591 |       53646 |       25523 |       24720 |       25221 |       24974 |       25306 |       24832 |       25090 |       25196 |       25014 |       26004 |       25089 |       24921 |       25270 |       25728 |       25108 |       25412 |       25515 |       25836 |       25291 |       25520 |       25452 |       25285 |       25347 |       26020 |       25300 |       25955 |       24981 |       25494 |       25606 |       25727
    num cache misses                 |    20498548 |        2591 |        1335 |        1275 |        1320 |        1272 |        1330 |        1281 |        1304 |        1300 |        1284 |        1312 |        1299 |        1289 |        1312 |        1313 |        1294 |        1318 |        1309 |        1325 |        1280 |        1312 |        1288 |        1313 |        1284 |        1322 |        1277 |        1308 |        1275 |        1279 |        1304 |        1296
    miss rate                        |       2.39% |       4.83% |       5.23% |       5.16% |       5.23% |       5.09% |       5.26% |       5.16% |       5.20% |       5.16% |       5.13% |       5.05% |       5.18% |       5.17% |       5.19% |       5.10% |       5.15% |       5.19% |       5.13% |       5.13% |       5.06% |       5.14% |       5.06% |       5.19% |       5.07% |       5.08% |       5.05% |       5.04% |       5.10% |       5.02% |       5.09% |       5.04%
    mpki                             |        4.86 |        6.14 |        6.98 |        6.86 |        6.98 |        6.79 |        7.03 |        6.87 |        6.93 |        6.89 |        6.85 |        6.80 |        6.92 |        6.89 |        6.94 |        6.86 |        6.88 |        6.94 |        6.87 |        6.90 |        6.78 |        6.89 |        6.78 |        6.94 |        6.79 |        6.84 |        6.76 |        6.77 |        6.80 |        6.72 |        6.83 |        6.76
  Cache L1-D                         |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
    num cache accesses               |  2814316016 |      145328 |       66243 |       63520 |       64397 |       63910 |       64520 |       63658 |       64133 |       64273 |       63887 |       65303 |       63938 |       63824 |       64242 |       64868 |       64035 |       64511 |       64687 |       65040 |       64130 |       64518 |       64517 |       64265 |       64271 |       65273 |       64144 |       65344 |       63889 |       65881 |       64814 |       66255
    num cache misses                 |      108012 |        3380 |        1807 |        1176 |        1294 |        1234 |        1298 |        1216 |        1257 |        1292 |        1280 |        1390 |        1321 |        1255 |        1267 |        1276 |        1267 |        1329 |        1269 |        1287 |        1313 |        1291 |        1266 |        1329 |        1280 |        1383 |        1271 |        1265 |        1331 |        1953 |        1309 |        1907
    miss rate                        |       0.00% |       2.33% |       2.73% |       1.85% |       2.01% |       1.93% |       2.01% |       1.91% |       1.96% |       2.01% |       2.00% |       2.13% |       2.07% |       1.97% |       1.97% |       1.97% |       1.98% |       2.06% |       1.96% |       1.98% |       2.05% |       2.00% |       1.96% |       2.07% |       1.99% |       2.12% |       1.98% |       1.94% |       2.08% |       2.96% |       2.02% |       2.88%
    mpki                             |        0.03 |        8.01 |        9.44 |        6.33 |        6.84 |        6.59 |        6.86 |        6.52 |        6.68 |        6.85 |        6.83 |        7.20 |        7.04 |        6.71 |        6.70 |        6.66 |        6.74 |        7.00 |        6.66 |        6.70 |        6.95 |        6.78 |        6.67 |        7.02 |        6.77 |        7.16 |        6.73 |        6.55 |        7.10 |       10.26 |        6.85 |        9.94
  Cache L2                           |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
    num cache accesses               |    20606915 |        6103 |        3199 |        2512 |        2722 |        2559 |        2688 |        2568 |        2610 |        2690 |        2661 |        2821 |        2690 |        2633 |        2673 |        2696 |        2667 |        2757 |        2676 |        2703 |        2659 |        2629 |        2606 |        2721 |        2648 |        2791 |        2599 |        2674 |        2697 |        3321 |        2668 |        3289
    num cache misses                 |       41524 |        3853 |        2694 |        2220 |        2382 |        2281 |        2360 |        2265 |        2308 |        2349 |        2330 |        2474 |        2375 |        2316 |        2347 |        2343 |        2335 |        2405 |        2328 |        2370 |        2359 |        2350 |        2332 |        2398 |        2327 |        2457 |        2325 |        2327 |        2366 |        2810 |        2362 |        2770
    miss rate                        |       0.20% |      63.13% |      84.21% |      88.38% |      87.51% |      89.14% |      87.80% |      88.20% |      88.43% |      87.32% |      87.56% |      87.70% |      88.29% |      87.96% |      87.80% |      86.91% |      87.55% |      87.23% |      87.00% |      87.68% |      88.72% |      89.39% |      89.49% |      88.13% |      87.88% |      88.03% |      89.46% |      87.02% |      87.73% |      84.61% |      88.53% |      84.22%
    mpki                             |        0.01 |        9.13 |       14.08 |       11.94 |       12.59 |       12.18 |       12.47 |       12.14 |       12.27 |       12.46 |       12.43 |       12.82 |       12.65 |       12.39 |       12.42 |       12.23 |       12.42 |       12.66 |       12.22 |       12.34 |       12.49 |       12.35 |       12.28 |       12.67 |       12.30 |       12.72 |       12.31 |       12.05 |       12.62 |       14.76 |       12.37 |       14.44
  Cache L3                           |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
    num cache accesses               |       41912 |        3987 |        2751 |        2281 |        2491 |        2334 |        2422 |        2335 |        2357 |        2447 |        2428 |        2593 |        2446 |        2404 |        2441 |        2450 |        2442 |        2515 |        2426 |        2461 |        2425 |        2376 |        2384 |        2477 |        2411 |        2543 |        2376 |        2428 |        2457 |        2900 |        2417 |        2857
    num cache misses                 |       35270 |        1576 |         525 |         580 |        1347 |        1329 |         824 |         962 |        1545 |        1152 |        1080 |         955 |        1340 |        1697 |         778 |         720 |        1128 |        1905 |         830 |         750 |        1355 |         607 |        1250 |        1249 |        1360 |        1282 |         695 |         982 |        1291 |        1936 |         825 |        1485
    miss rate                        |      84.15% |      39.53% |      19.08% |      25.43% |      54.07% |      56.94% |      34.02% |      41.20% |      65.55% |      47.08% |      44.48% |      36.83% |      54.78% |      70.59% |      31.87% |      29.39% |      46.19% |      75.75% |      34.21% |      30.48% |      55.88% |      25.55% |      52.43% |      50.42% |      56.41% |      50.41% |      29.25% |      40.44% |      52.54% |      66.76% |      34.13% |      51.98%
    mpki                             |        0.01 |        3.73 |        2.74 |        3.12 |        7.12 |        7.10 |        4.35 |        5.16 |        8.22 |        6.11 |        5.76 |        4.95 |        7.14 |        9.08 |        4.12 |        3.76 |        6.00 |       10.03 |        4.36 |        3.90 |        7.18 |        3.19 |        6.58 |        6.60 |        7.19 |        6.64 |        3.68 |        5.09 |        6.89 |       10.17 |        4.32 |        7.74
DRAM summary                         |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
  num dram accesses                  |        7729 |           0 |           0 |           0 |        7716 |           0 |           0 |           0 |        7541 |           0 |           0 |           0 |        7405 |           0 |           0 |           0 |        7356 |           0 |           0 |           0 |        7304 |           0 |           0 |           0 |        7259 |           0 |           0 |           0 |        7635 |           0 |           0 |           0
  average dram access latency (ns)   |       54.50 |         inf |         inf |         inf |       54.69 |         inf |         inf |         inf |       54.55 |         inf |         inf |         inf |       54.41 |         inf |         inf |         inf |       54.40 |         inf |         inf |         inf |       54.31 |         inf |         inf |         inf |       54.39 |         inf |         inf |         inf |       54.37 |         inf |         inf |         inf
  average dram queueing delay        |        1.07 |        0.00 |        0.00 |        0.00 |        1.26 |        0.00 |        0.00 |        0.00 |        1.12 |        0.00 |        0.00 |        0.00 |        0.98 |        0.00 |        0.00 |        0.00 |        0.97 |        0.00 |        0.00 |        0.00 |        0.88 |        0.00 |        0.00 |        0.00 |        0.96 |        0.00 |        0.00 |        0.00 |        0.94 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |             |            
  num loads from dram                |        3993 |         277 |          47 |          46 |          82 |          28 |          62 |          35 |         200 |         184 |         104 |          73 |         168 |         236 |          76 |          38 |         124 |         288 |          61 |          55 |         193 |          38 |         110 |         199 |         162 |         212 |          43 |          81 |         166 |         188 |         102 |         117
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |         233 |         162 |          26 |          69 |         269 |         268 |         158 |         183 |         130 |         153 |         187 |         216 |         190 |         147 |         165 |         174 |         187 |         178 |         138 |         154 |         168 |          94 |         133 |         151 |         123 |         153 |         106 |         141 |         178 |         192 |         153 |         154
